netlify.toml

```diff
@@ -4,4 +4,8 @@
 [[redirects]]
   from = "/*"
   to = "/error"
-  status = 404
\ No newline at end of file
+  status = 404
+
+[[redirects]]
+  from = "/articles/blog/:splat"
+  to = "/blog"
```

Comment:
Still not working, I actually not understand why it's still not working, already follows this https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file and this https://docs.netlify.com/routing/redirects/redirect-options/